main.lua

```diff
@@ -55,8 +55,11 @@ end
 function startServers()
 	local wd, _ = os.Getwd()
 	rootUri = fmt.Sprintf("file://%s", wd)
-	local envFallback, _ = os.Getenv("MICRO_LSP")
-	local server = mysplit(config.GetGlobalOption("lsp.server") or envFallback or '', ",")
+	local fallback, _ = os.Getenv("MICRO_LSP")
+	if ("" == fallback) then
+		fallback = 'python=pylsp,go=gopls,typescript=deno,rust=rls,lua=lua-lsp lsp={"importMap": "./import_map.json"}'
```

Comment:
The part with `={"importMap":"./import_map.json"}` is just an example of how to transfer additional initialization options to an LSP. For a fallback, I would leave it out. 

The above line should be the following instead:

```
  fallback = 'python=pylsp,go=gopls,typescript=deno lsp,javascript=deno lsp,rust=rls,lua=lua-lsp'
```

I have to try out to see if any error messages would be shown if an LSP is not actually there.

Reply:
Luckily if a language server does not exist, it seems it just gracefully says "onExit <nil>" in the log.txt and no error message is shown in the UI, so this is good! Seems like only the fallback itself needs to be adapted :)